### PR TITLE
Some new features

### PR DESCRIPTION
**Pull request by ghost**
Added buffer.clear() and buffer.fill()

### CODE REVIEW

**Comment by tomsmeding** [comment]
Hi, thanks for the ideas :)

I have to ask though: why remove all the auxiliary files and the test set? I really fail to see how that's relevant to this PR.

 What would you use `clear` and `fill` for, in a setting where you want a circular buffer? i.e. what use case would this help?

File: README.md [hunk 1]
```diff
@@ -52,9 +56,9 @@ buf.deq(); // -> throws RangeError("CircularBuffer dequeue on empty buffer")
   - Get the value at index `idx`. `0` is the front of the buffer (last-enqueued item, or first-pushed item), `buf.size()-1` is the back of the buffer.
 - `get(start,end)` -> `[value]`
   - Gets the values from index `start` up to and including index `end`; returns an array, in front-to-back order. Equivalent to `[buf.get(start),buf.get(start+1),/*etc*/,buf.get(end)]`.
+- `fill(value)` -> `undefined
```
Comment (lieuwex) [inline README.md]: Missing a backtick here

File: README.md [hunk 2]
```diff
@@ -52,9 +56,9 @@ buf.deq(); // -> throws RangeError("CircularBuffer dequeue on empty buffer")
...
+  - Fills the buffer with elements of selected value
+- `clear()` -> `undefined
```
Comment (lieuwex) [inline README.md]: And missing a backtick here

**Comment by tomsmeding** [comment]
I'm sorry, I get the feeling that you're not sure how git(hub?) works; if you are, then I'm at a loss as to what you're doing. Please come back when you have a useful contribution.

**Pull request closed without merging**